nrf_security/doc/backend_config.rst

```diff
@@ -36,28 +34,21 @@ If only a subset of the backends supports a given feature, this information is p
 AES configuration
 *****************
 
-AES core support can be enabled by setting the :kconfig:`CONFIG_MBEDTLS_AES_C` Kconfig variable.
-Enabling AES core support enables AES ECB cipher mode and allows for the following ciphers to be configured: CTR, OFB, CFB, CBC, XTS, CMAC, CCM/CCM*, and GCM.
+The AES core is enabled with the Kconfig option :kconfig:`CONFIG_MBEDTLS_AES_C`.
+This enables AES ECB cipher mode and allows the ciphers CTR, OFB, CFB, CBC, XTS, CMAC, CCM/CCM*, and GCM to be configured.
```

Comment:
There is one additional fixup that can be made. Our statement "ciphers" for all of this is prabably a bit broad. Especially on CCM/GCM which are AEAD algorithms that use the AES block ciphers.

Suggestion: Change to `ciphers and modes` as this is something that works for all